build/prbuild.yml

```diff
@@ -8,6 +8,14 @@ variables:
   SignAppForRelease: 'false'
 
 jobs:
+- template: ui-test-job.yml
```

Comment:
We could put the UI tests after the unit tests instead if that makes more sense

Reply:
They'll all run in parallel, but the proposed ordering seems simpler from a "mental organization" perspective. Can we also flag the unit test jobs as unit tests, instead of just Debug and Release?

---
In reply to: [303156978](https://github.com/microsoft/accessibility-insights-windows/pull/428#discussion_r303156978) [](ancestors = 303156978)

Reply:
pending